source/_remoteClient/input.py

```diff
@@ -217,15 +125,15 @@ def sendKey(vk: int | None = None, scan: int | None = None, extended: bool = Fal
 	:param extended: Whether this is an extended key, defaults to False
 	:param pressed: ``True`` if key pressed; ``False`` if released, defaults to True
 	"""
-	i = INPUT()
-	i.union.ki.wVk = vk
+	i = user32.INPUT()
```

Comment:
could we rename this to 

```suggestion
	input = user32.INPUT()
```

Reply:
Done